### DISCUSSION TITLE
Editor features profiles should be per Project instead of global

### DISCUSSION THREAD

**Discussion by mrkdji** [godotengine/godot-proposals#6016] [Editor]
I often tweak the _Manage Editor Features_ panel so that I have an easier time adding new nodes or creating new resources, but as I am working on multiple projects at once I find myself switching between feature profiles quite often. 
I think it would be more sensible to make this configuration per project instead of global, what do you think?

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/827.
